### PR TITLE
Adding encoding backward compatibility

### PR DESCRIPTION
**Pull request by pedrofaria**

### CODE REVIEW

File: server/import.go [hunk 1]
```diff
@@ -46,14 +47,24 @@ func ImportServer(configuration, settings, event, eventRules, entrylist, bop, as
...
-func importFile(reader io.Reader, config interface{}, filename string) error {
+func importFile(reader io.ReadSeeker, config interface{}, filename string) error {
 	r := transform.NewReader(reader, utf16Encoding.NewDecoder().Transformer)
 
 	if err := json.NewDecoder(r).Decode(config); err != nil {
-		logrus.WithError(err).
-			WithField("file", filename).
-			Error("Error unmarshalling configuration file JSON on import")
-		return err
+		if _, er := reader.Seek(0, io.SeekStart); er != nil {
```
Comment (Kugelschieber) [inline server/import.go:54]: Always name errors "err" if possible.

File: server/import.go [hunk 2]
```diff
@@ -46,14 +47,24 @@ func ImportServer(configuration, settings, event, eventRules, entrylist, bop, as
...
+			return er
+		}
+
+		// trying decode non ut16 content
+		if err2 := json.NewDecoder(reader).Decode(config); err2 != nil {
```
Comment (Kugelschieber) [inline server/import.go:59]: You can just redeclare "err" here. The `if err := ...` construct has its own scope.

File: server/import.go (hunk 1 above)
Comment (pedrofaria) [reply server/import.go:54]: I don't want to overwrite the main err

File: server/import.go (hunk 2 above)
Comment (pedrofaria) [reply server/import.go:59]: If I do it, I will loose the main error

**Review by Kugelschieber — APPROVED** [review]

**Review by itsmylife44 — APPROVED** [review]

**Pull request merged**